pom.xml

```diff
@@ -31,6 +31,15 @@
     <relativePath /> <!-- lookup parent from repository -->
   </parent>
 
+  <repositories>
+    <!-- Including for fontawesome -->
+    <repository>
+      <id>spring-repo</id>
+      <name>Spring Repository</name>
+      <url>https://repo.spring.io/plugins-release/</url>
+    </repository>
+  </repositories>
```

Comment:
shoul use official binaries at https://bintray.com/jerady/maven/FontAwesomeFX  think spring repo is just a link to that but better save than sorry

```suggestion
 <repositories>
    <!-- Including for fontawesome -->
    <repository>
      <id>fontawesomefx-repo</id>
      <name>FontAwesome Repository</name>
      <url>https://bintray.com/jerady/maven/FontAwesomeFX</url>
    </repository>
  </repositories>
```

Reply:
Can you please have a look on if we can tell mvn that this repo should just be used for the fontawesome libs?

Reply:
this repository is used only if dependency is not available in cxentral maven repository